### ISSUE TITLE
508-defect-2 [COGNITION]: Appointment date on the Review your appointment details screen must be a heading level 3

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#24170]
# [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** _First name only. Delete if not applicable._

## Details

The appointment date is using a `<strong>` tag while the VAMC title and Your contact details are using H3s. The date must be an H3 also so it retains semantic meaning like the other two section headings. Screenshot attached.

## Acceptance Criteria

- [ ] Visual styling is retained
- [ ] The date "Thursday, May 13..." announces as a "heading level three" on screen readers
- [ ] 0 axe-core violations

## Environment

* https://staging.va.gov/health-care/schedule-view-va-appointments/appointments/new-covid-19-vaccine-booking/review

## WCAG or Vendor Guidance (optional)

* https://www.w3.org/TR/UNDERSTANDING-WCAG20/content-structure-separation-programmatic.html
* https://www.tpgi.com/heading-off-confusion-when-do-headings-fail-wcag/

## Screenshots or Trace Logs

![Screen Shot 2021-05-04 at 9 48 35 AM](https://user-images.githubusercontent.com/934879/117023052-5da83300-acbe-11eb-81f7-88e7757bea28.png)

**Comment by outerpress** [comment]
@1Copenut looking at this one, I don't think it makes sense for this to be an h3 (or bold) as it's not a heading for any additional information.  I think it may work better styled as default copy under the COVID-19 vaccine header, without the divider. I'll mock this up.

**Comment by 1Copenut** [comment]
@outerpress I feel we do need to make this an H3. It's not a heading for additional information, but it carries the same visual weight as two text blocks that are headings. It may be a slight semantic overload, but increases the chance that information is heard by a screen reader and doesn't cost us anything visually.

**Comment by outerpress** [comment]
>  It may be a slight semantic overload, but increases the chance that information is heard by a screen reader and doesn't cost us anything visually.

Gotcha, I thought strong text was called out in VO. Thanks! @laurenernest this one's GTG from my perspective.

**Comment by laurenernest** [comment]
Great! Moving to dev ready.
